testing/python/metafunc.py

```diff
@@ -1415,3 +1415,31 @@ def test_func(x):
             "*test_func*0*PASS*",
             "*test_func*2*PASS*",
         ])
+
+    _make_parametrize_return_conf_blob = """
```

Comment:
it seems like this could make a nice class level autouse fixture to make the shared code nicer